### PR TITLE
Add the read and write irmin benchmark

### PR DESCRIPTION
**Pull request by shubhamkumar13**
Add the basic_read_write file in the irmin benchmark

### CODE REVIEW

File: run_config.json [hunk 1]
```diff
@@ -1113,7 +1113,10 @@
       "ismacrobench": false,
```
Comment (kayceesrk) [inline run_config.json]: Can you rename this to `irmin_mem_rw` instead of `basic_read_write`

Comment (shubhamkumar13) [reply run_config.json]: Yes sure, that seems more fitting.

Comment (kayceesrk) [reply run_config.json]: The commit days comment out necessary print statements. But I don’t see any print statements commented out. Can you ensure that those files are not missed out.

Comment (shubhamkumar13) [reply run_config.json]: I checked the files and they are not missed out.
I made a mistake in the commit message, those print statements were present on the https://github.com/ocaml-bench/sandmark/pull/98/commits/56e64294ab9d33f067b9c12ce52c8ebe5f51eb95 commit. At that time I was probably thinking of squashing them
I'm renaming the most recent commit message.

**Pull request merged**